contributor-guide/modules/ROOT/pages/build-systems/b2.adoc

```diff
@@ -1,5 +1,33 @@
 = B2
+:navtitle: B2
 
-*Content under construction.*
+B2 is a high-level build system, developed to use with Boost, designed to make it easy to build pass:[C++] projects. Building projects using B2 often involves no more than a single command line. This section introduces what library developers need to know about preparing a library for B2.
```

Comment:
> developed to use with Boost

Although a lot of people will describe it this way, Rene will definitely complain about this.

Reply:
removed the phrase as it doesn't add much value